### PR TITLE
Formatting changes

### PR DESCRIPTION
**Pull request by spiray**

### CODE REVIEW

**Review by ewenjo — APPROVED** [review]

File: src/api/faq/faq.controller.js [hunk 1]
```diff
@@ -8,20 +8,14 @@ async function getFAQ(req, res, next) {
...
     } else response = await FAQ.find({}).exec();
 
     const finalResponse = response
       .map(({
-        id,
-        question,
-        answer,
-        createdAt,
-        updatedAt,
+        id, question, answer, createdAt, updatedAt,
```
Comment (w3cj) [inline src/api/faq/faq.controller.js:18]: I prefer the newlines for imports and destructuring. I will try to find a linter rule or prettier rule to enforce this.

**Review by w3cj — CHANGES_REQUESTED** [review]
See comment above. I prefer destructured items to be on a new line.

File: src/api/faq/faq.controller.js (hunk 1 above)
Comment (spiray) [reply src/api/faq/faq.controller.js:18]: OK that would be good. They were just getting messy and inconsistent.

Comment (ewenjo) [reply src/api/faq/faq.controller.js:18]: It would be nice to have specific rules on how the code should be formatted.

**Pull request closed without merging**